samples/drivers/led_pwm/sample.yaml

```diff
@@ -6,3 +6,4 @@ tests:
     filter: dt_compat_enabled("pwm-leds")
     tags: LED
     depends_on: pwm
+    platform_exclude: reel_board
```

Comment:
Hi, is this the solution we decided to apply for this issue already? 

If not, my suggestion is we can add harness+regex in sample.yaml to catch messages and judge it.
Although we cannot automatically know whether led works correctly or not (seems like only judge by our eyes), we can check if there is any error message happened from the code to judge it at least.

Reply:
There's no errors from this sample. What it does is just implementations of LED APIs for PWM driven LEDs without any testcases. It's valid and only tested on STM32F070-based boards per [29401](https://github.com/zephyrproject-rtos/zephyr/pull/29401), but its only filter is "pwm" dependency. Therefore, the reason it appears as an error in the daily test is simply due to bad filtering. If it fails on multiple boards in the daily test then there'll be a strong need to use regex to catch the output.

Reply:
Ok, I understood. And just sync with Steven this is the solution we will apply for this. Thanks!